doc/architecture/frontend.md

```diff
@@ -0,0 +1,57 @@
+# Frontend Architecture
+
+## Structure
+
+- `__mocks__/`: Contains mocked modules for tests
+- `assets/`: Contains asset files like images
+- `bindings/`: Contains the generated API bindings
+- `components/`: Contains components that are shared across features
+- `config/`: Contains initialization code for the environment and libraries
+- `features/`: Contains the distinct features of the application
+  - `dark_mode/`: Contains the dark_mode feature (Switcher and store)
+  - `imprint_page/`: Contains the imprint page
+  - `landing_page/`: Contains the landing page
+  - `map_planning/`: Contains the map with layers
+  - `seeds/`: Contains the seed routes
+- `generated/`: Contains generated code, for now only docs
```

Comment:
Why isn't bindings also a subfolder here? Here description of subfolders might make sense, as the list doesn't change quickly.